### PR TITLE
Respawn Waves & Force Respawn Enable

### PR DESCRIPTION
**Pull request by dill0nRoberts**
Configurable Respawns, forced respawn to be on, minimum respawn timer

### CODE REVIEW

**Comment by ghost** [comment]
Please do not merge this just yet, i have some suggestions but I'm currently on the train, will reply later.

**Comment by dill0nRoberts** [comment]
Not sure how to merge, main thing is that respawn needs to be default (In all configs it is enabled but something sets it to 0 when server starts) enabled since we don't have full admin team yet, I'll be away for a few hours today and just wanted to make it accessible to everyone while I'm gone. Make whatever changes you see fit, I know you're more knowledgeable than I am about all this.

File: code/modules/mob/mob.dm [hunk 1]
```diff
@@ -44,11 +44,14 @@ var/next_mob_id = 0
 
 	usr.show_message(t, 1)
 
-/mob/proc/show_message(msg, type, alt, alt_type)//Message, type of message (1 or 2), alternative message, alt message type (1 or 2)
+/mob/proc/show_message(msg, type, alt, alt_type)
+//Message, type of message (1 or 2), alternative message, alt message type (1 or 2)
+//^^Highly fucking useful commenting there yogi bear^^
+//TODO: Switch these stupid fucking numbers with predefined literals (i.e #define MESSAGE1TYPE 1)
```
Comment (ghost) [inline code/modules/mob/mob.dm]: No need for these useless comments.

File: code/modules/mob/mob.dm [hunk 2]
```diff
@@ -44,11 +44,14 @@ var/next_mob_id = 0
...
 
 	if(!client)	return
 
-	msg = copytext(msg, 1, MAX_MESSAGE_LEN)
+	msg = copytext(msg, 1, MAX_MESSAGE_LEN) // good job einstein
```
Comment (ghost) [inline code/modules/mob/mob.dm]: No need for these useless comments.

File: code/modules/mob/mob.dm [hunk 3]
```diff
@@ -465,13 +468,19 @@ var/list/slot_equipment_priority = list( \
 /mob/verb/abandon_mob()
 	set name = "Respawn"
 	set category = "OOC"
-
-	if (!( abandon_allowed ))
+	if(!config.respawn_FORCE)
```
Comment (ghost) [inline code/modules/mob/mob.dm]: No need for this, abandon_allowed can be set trough the config.

Comment (dill0nRoberts) [inline code/modules/mob/mob.dm]: It seems to be (enabled), but something disables it, I will try to find where
Update: Found it

**Comment by ghost** [comment]
I'm unable to pull this, as it still adds the not-needed force respawn option.

**Pull request merged**